fiboa_cli/datasets/be_vlg.py

```diff
@@ -0,0 +1,85 @@
+from ..convert_utils import convert as convert_
+
+URI = "https://www.landbouwvlaanderen.be/bestanden/gis/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024)_GPKG.zip"
+# in the zipfile, the gpkg has a different name so we need to add this while opening the file
+PATH_TEMPLATE = "/vsizip/{}/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024).gpkg"
```

Comment:
That's an interesting idea, let me test this on Windows and with some other converters where we need this, too (and solved it with just extracting from ZIP manually), e.g. at and the eurocrops datasets.

Reply:
I ran `fiboa convert be_vlg -o be -collection -c be.gpkg` and it returned

> '/vsizip/be.gpkg/Landbouwgebruikspercelen_2023_-_Definitief_(extractie_28-03-2024).gpkg' does not exist in the file system, and is not recognized as a supported dataset name. I guess this doesn't quite work with the -c option yet?

Reply:
To unblock this PR I've changed the code to use the same manual unzipping approach as in the other converters for now. I'll follow-up with a better solution through issue #47.